api/auth.py

```diff
@@ -20,12 +20,22 @@ class TokenData(BaseModel):
     username: Optional[str] = None
 
 
+class Settings(BaseSettings):
+    secret_key: str
+
+    class Config:
+        env_file = "../.env"
```

Comment:
This can be dropped, the environment file will be loaded by docker-compose and its contents will be exported as environment variables.  So when the container is started, an environment variable `SECRET_KEY` should already be there.  Then the FastAPI settings will read that and populate the `Settings.secret_key` value.

See the documentation here: https://fastapi.tiangolo.com/advanced/settings/